drivers/ipm/Kconfig.template.nrf_ipc_channel

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+
+menu "IPM Message Channel [$(nrfx_ipc_num)] configuration"
+
+config IPM_MSG_CH_$(nrfx_ipc_num)_ENABLE
+	bool "Enable IPM Message Channel $(nrfx_ipc_num)"
+
+choice
+	prompt "IPM Message Channel $(nrfx_ipc_num) direction"
+	default IPM_MSG_CH_$(nrfx_ipc_num)_RX
+	depends on IPM_MSG_CH_$(nrfx_ipc_num)_ENABLE
+
+config IPM_MSG_CH_$(nrfx_ipc_num)_RX
+	bool "RX"
+
+config IPM_MSG_CH_$(nrfx_ipc_num)_TX
+	bool "TX"
+
+endchoice
```

Comment:
Would changing RX to a bool, and having

config IPM_MSG_CH_$(nrfx_ipc_num)_TX
  default ! IPM_MSG_CH_$(nrfx_ipc_num)_RX

help with your issues with choice?

Reply:
Do you mean something along these lines?

```
menu "IPM Message Channel [$(nrfx_ipc_num)] configuration"

config IPM_MSG_CH_$(nrfx_ipc_num)_ENABLE
	bool "Enable IPM Message Channel $(nrfx_ipc_num)"

config IPM_MSG_CH_$(nrfx_ipc_num)_RX
	bool "IPM Message RX Channel"
	depends on IPM_MSG_CH_$(nrfx_ipc_num)_ENABLE

config IPM_MSG_CH_$(nrfx_ipc_num)_TX
	bool "IPM Message TX Channel"
	depends on IPM_MSG_CH_$(nrfx_ipc_num)_ENABLE
	default ! IPM_MSG_CH_$(nrfx_ipc_num)_RX

endmenu
```

Yes, getting rid of choice option solves issues with choice :) and I can simply select proper configuration.

Reply:
I think RX and TX could both get enabled with that configuration. I was thinking of having one option promptless to prevent this. But that might not allow you to select from other options as you want.

So I don't really see a clean solution here.

Reply:
Yes, I am going to use my suggestion in this PR. It could be improved in other PR if we find a cleaner way of doing it